src/tasks/models/task_result.py

```diff
@@ -79,6 +82,7 @@ def save(self):
                 disk_space = shutil.disk_usage(data_path)
                 percent_used = (disk_space.used / disk_space.total) * 100
                 if percent_used > self.max_disk_usage:
+                    logger.info("Max disk usage exceeded, deleting oldest task result!")
```

Comment:
Suggest changing this to warning log level

Reply:
Fixed in cf6df43